clients/bazel-module-registry/src/main/kotlin/Model.kt

```diff
@@ -80,6 +83,19 @@ class GitRepositorySourceInfo(
     val stripPrefix: String? = null
 ) : ModuleSourceInfo()
 
+/**
+ * A module source info with type 'local_path'.
+ */
+@Serializable
+@SerialName("local_path")
+class LocalRepositorySourceInfo(
+    val path: String,
+    // The following variable is not naturally present in the source.json specification: it will be populated by the
```

Comment:
Is "naturally" correct here? Do you mean "actually"? Maybe better say "is not part of the ..."?

Nits: Put `sources.json` into backticks or quotes, and start capitalized after the colon.